### ISSUE TITLE
Tool's limitation?

### ISSUE THREAD

**Issue by potoo0** [RaMMicHaeL/Textify#1]
Thanks for your tool, but I found a limitation ? : cannot copy text from sublime text 3 's error. Here its a snapshot of it.

![snipaste_2018-07-30_20-08-31](https://user-images.githubusercontent.com/34411681/43396521-63801810-9434-11e8-8f4f-678ce127e704.png)

**Comment by RaMMicHaeL** [comment]
Textify relies on UI Automation. There’s a tool from Microsoft called Inspect which shows automation-related information. It can be downloaded [here](https://www.dropbox.com/s/5cfa754k000zt07/inspect.exe?dl=1).

Please try it on the text which Textify can’t handle, and let me know whether Inspect is able to extract the text.

Also note that if Sublime Text is running as administrator and Textify doesn't, the mouse shortcut won't work. The keyboard shortcut should work, though - it's `Ctrl+Shift+Alt+T` by default.

**Comment by potoo0** [comment]
That issue cannot be test, cause I figured that error, sorry for that. 

I test other error of sublime text 3 in same windows user, this tool works !(I didn't download Inspect tool)
![snipaste_2018-07-31_10-25-07](https://user-images.githubusercontent.com/34411681/43434422-1eff2f58-94ae-11e8-9dc5-5970473c1986.png)

If I found some text which this tool can't handle, I will notify you immediately to promote improvement.
Thanks for your tool makes computer more efficient.
